hazelcast/src/test/resources/com/hazelcast/client/console/testjob-with-hazelcast-codebase/src/main/java/com/hazelcast/jet/testjob/TestJob.java

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
This test job uses deprecated jet APIs. Could you use this one: https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/test/resources/com/hazelcast/client/console/testjob-with-hz-bootstrap/src/main/java/com/hazelcast/jet/testjob/TestJob.java instead so that we don't have to update this test when that deprecated staff is removed.

Reply:
Fixed: https://github.com/hazelcast/hazelcast/pull/19512/commits/a2878f5d253193e458922705b5b21d6b18242ed5